html/datasets.html

```diff
@@ -0,0 +1,313 @@
+<!DOCTYPE html>
+<html>
+<head lang="en">
+    <meta charset="UTF-8">
+    <title>Dataset Browser</title>
+
+    <script type="text/javascript" src="facetview2/vendor/jquery/1.7.1/jquery-1.7.1.min.js"></script>
```

Comment:
We need to change this to what's going to be in production (basically the commented out section below). When we push this it's what we're pushing into production.